### PR TITLE
Spaces Code

### PR DESCRIPTION
**Pull request by ravensears**

### CODE REVIEW

File: coverage/.resultset.json [hunk 1]
```diff
@@ -22,44 +22,69 @@
           1,
```
Comment (Haydnstokes92) [inline coverage/.resultset.json]: maybe add this file to git ignore

File: spec/features/new_space_spec.rb [hunk 2]
```diff
@@ -8,5 +8,16 @@
...
-    end
-end
\ No newline at end of file
+  end
+
+  scenario 'printing all spaces' do
+    visit '/spaces/new' 
+
+    fill_in 'name', with: 'House on the Hills'
+    fill_in 'description', with: '12 bedroom with own private beach'
```
Comment (naima90) [inline spec/features/new_space_spec.rb:17]: love the description

File: spec/space_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      data = persisted_data(space.id)
+
+      expect(space).to be_a Space
+      expect(space.id).to eq data['id']
+      expect(space.name).to eq "Cher's house"
+      expect(space.description).to eq "It's fab"
+      expect(space.price).to eq "2000"
+    end
+  end
+end
```
Comment (Haydnstokes92) [inline spec/space_spec.rb:18]: really nice, easy to read and understandable code

**Review by naima90 — COMMENTED** [review]
looks really good, just need to figure out the conflict with the files

**Pull request merged**